src/controllers/project.js

```diff
@@ -383,13 +386,87 @@ const getProjectProfile = async (req, res, next) => {
   return next();
 };
 
+const client = elasticsearch.Client({
+  host: process.env.ELASTICSEARCH_URL,
+});
+
+const indexProjects = async (req, res) => {
+  const projects = await Project.find({});
```

Comment:
when will this indexing happen, why aren't we indexing with each newly added project? and removing projects from the index during a `delete` request?

Reply:
this is a separate ep, once we need we can manually index the projects by sending a request here